h2integrate/tools/eco/utilities.py

```diff
@@ -1709,7 +1714,15 @@ def calculate_lca(
     ]  # GHG Emissions Intensity of supplying Lime to processes accounting for limestone mining,
     # lime production, lime processing, and lime transportation assuming 20 miles via Diesel engines
     # (kg CO2e/kg lime)
-    # ------------------------------------------------------------------------------
+    # # ------------------------------------------------------------------------------
+    # # Carbon Coke
+    # # ------------------------------------------------------------------------------
+    # coke_supply_EI = greet_data_dict[
+    #     "coke_supply_EI"
+    # ]  # GHG Emissions Intensity of supplying Coke to processes accounting for combustion
+    # # and non-combustion emissions of coke production
+    # # (kg CO2e/kg Coke)
+    # # ------------------------------------------------------------------------------
```

Comment:
Non-blocking nitpick: does this need to remain if it's commented out? Will it be useful later?